### PR TITLE
Added useResizable sample in qa-sampler

### PR DESCRIPTION
**Pull request by juwonjeong**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added useResizable sample in qa-sampler

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-5710

### Comments
Related to https://github.com/enactjs/enact/tree/feature/WRO-5710

### CODE REVIEW

File: samples/sampler/stories/qa/Resizable.js [hunk 1]
```diff
@@ -2,13 +2,20 @@ import Button from '@enact/sandstone/Button';
 import Item from '@enact/sandstone/Item';
 import Scroller from '@enact/sandstone/Scroller';
 import Resizable from '@enact/ui/Resizable';
+import useResizable from '@enact/ui/Resizable/useResizable';
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/Resizable.js:5]: If we refactor `Resizable` to use `useResizable` inside of it, I think we don't need to modify this sampler.

Comment (juwonjeong) [reply samples/sampler/stories/qa/Resizable.js:5]: I updated `Resizable` to use `useResizable`.

Comment (MikyungKim) [reply samples/sampler/stories/qa/Resizable.js:5]: I meant if we modify `ui/Resizable` to use `useResizable`, we don't need this PR anymore. If that case, please close this PR.
:)

Comment (juwonjeong) [reply samples/sampler/stories/qa/Resizable.js:5]: Okay, I wil close this PR.

**Pull request closed without merging**